src/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import _ from 'lodash';
+/* eslint-enable */
```

Comment:
### [ OPTIONAL]

- [ ] This change should have been requested by the previous reviewer. It is always good to fix linter errors instead of disabling them.